tests/test_6707_unit.py

```
@@ -0,0 +1,126 @@
+        return self._attrs.get(key, _NO_MATCH_ID if key.endswith("_id") else _NO_MATCH_IOU)
+    def __setitem__(self, key, value):
+        self._attrs[key] = value
+    def get(self, key, default=None):
+        return self._attrs.get(key, default)
+    def get_attribute_value(self, key, default=None):
+        return self._iscrowd if key == "iscrowd" else default
+
+
+def _make_cats(gt_specs, pred_specs):
+    id_key, iou_key = "eval_id", "eval_iou"
+    gts   = [FakeDet(l, i, b) for l, i, b in gt_specs]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)